docs/developers/RELEASE.md

```diff
@@ -20,9 +20,13 @@ to add it as a secret.
 
 ## determining the version
 
-The version of `napari` is automatically determined by [`versioneer`](https://github.com/warner/python-versioneer)
-from the latest [`git` tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) beginning with `v`.
-Thus, you'll need to tag the [reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References) with the new version number. It is likely something like `X.Y.Z`. Before making a release though we need to generate the release notes.
+The version of `napari` is automatically determined at install time by
+[`setuptools_scm`](https://github.com/pypa/setuptools_scm) from the latest
```

Comment:
I'm not sure you meant to include this change, at least while https://github.com/napari/napari/pull/1287 is still under discussion

Reply:
I mentioned in the original post here:

> Note: this PR also includes changes from #1287, since they were required to get mac working... but if that proposal is rejected, I can undo it here and figure out another way.

So yeah, the entire #1287 is in here... and if that merges it will be disappear from this diff.  If it _doesn't_ merge, then we have to figure out how to get around some versioning issues.  But since it simplifies everything for us here, i'd prefer to leave it in until we know...

Reply:
Oops I missed that, sorry